ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -177,6 +209,68 @@ + (void)configureDocumentViewController:(PTDocumentViewController*)documentViewC
                 else if ([key isEqualToString:PTMultiTabEnabledKey]) {
                     // Handled by tabbed config.
                 }
+                else if ([key isEqualToString:PTFitModeKey]) {
+                    id fitModeValue = configPairs[PTFitModeKey];
+                    if (![fitModeValue isEqual:[NSNull null]])
+                    {
+                        NSAssert([fitModeValue isKindOfClass:[NSString class]], @"fitMode not in expected string format.");
+                        if ([fitModeValue isKindOfClass:[NSString class]]) {
+                            NSString* fitMode = fitModeValue;
+                            [(PTFlutterViewController *)documentViewController setFitMode:fitMode];
+                        }
+                        else
+                        {
+                            NSLog(@"fitMode not in expected string format.");
```

Comment:
A bit concerning that errors are not passed out programatically. Could we do so using the error pattern described above?